### DISCUSSION TITLE
opentelemetry monorepos include contrib

### DISCUSSION THREAD

**Discussion by michaeldop** [renovatebot/renovate#21830] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

I would like to also include the contrib repos in the [opentelemetry monorepo presets](https://github.com/renovatebot/renovate/blob/main/lib/config/presets/internal/monorepo.ts#L172-L175) . Should I just open a PR to make this the default behavior?

for now I have just been including this in my renovate.json file for the go Monorepo
```
       {
            "extends": ["group:opentelemetry-goMonorepo"],
            "matchSourceUrls": [
                "https://github.com/open-telemetry/opentelemetry-go-contrib"
            ]
        }
```

**Comment by rarkins** [comment]
@secustor does this make sense?

**Reply by secustor** [reply]
There is a hard dependency from contrib repos to there upstream counter parts.

The most sense for makes it to create a `group:opentelemetryContribs` which then does this for all sdks.
